integr8ly/installation
# Updating webapp walkthrough locations

## Additional Information
- Updating default Webapp walkthrough locations
- Resetting and renaming upgrade task set in webapp role
- Adding upgrade logic to patch CR with newly updated walkthrough locations in webapp upgrade task

## Verification Steps
As the verifier of the PR the following process should be done:

1. Complete an install of RHMI from this branch
2. Login to Solution Explorer
3. Navigate to 'Managing and protecting APIs using API management' solution pattern
4. In the right hand nav, click the 'Launcher Documentation' link - this should direct to https://appdev.openshift.io/docs/

### Installation Verification

1. Perform an install against a target Openshift cluster and follow the verification steps listed above.

### Upgrade Verification

1. Install `release-1.7.1` on a target Openshift cluster
2. Follow the verification steps above but in this scenario the 'Launcher Documentation' link should bring you to a broken link
3. Run an upgrade against the same cluster and follow the verification steps listed above. This time the 'Launcher Documentation' link should work i.e. point to https://appdev.openshift.io/docs/

## Checklist



Is an upgrade task required and are there additional steps needed to test this?
